lib/kafka.ts

```diff
@@ -49,29 +49,32 @@ export const createKafkaInstance = async function (page: Page, name: string, che
 export const deleteKafkaInstance = async function (page: Page, name: string, awaitDeletion = true) {
   const instanceLinkSelector = page.getByText(name);
   const row = page.locator('tr', { has: instanceLinkSelector });
-
-  await row.locator('[aria-label="Actions"]').click();
-  await page.locator('button', { hasText: 'Delete instance' }).click();
   try {
-    await expect(page.locator('input[name="mas-name-input"]')).toHaveCount(1, { timeout: 5000 });
-
-    // FIXME: workaround for https://github.com/redhat-developer/app-services-ui-components/issues/590
-    // https://github.com/microsoft/playwright/issues/15734#issuecomment-1188245775
-    await new Promise((resolve) => setTimeout(resolve, 500));
-    await page.locator('input[name="mas-name-input"]').click();
-
-    await page.locator('input[name="mas-name-input"]').fill(name);
+    await row.locator('[aria-label="Actions"]').click();
+    await page.locator('button', { hasText: 'Delete instance' }).click();
+    try {
+      await expect(page.locator('input[name="mas-name-input"]')).toHaveCount(1, { timeout: 5000 });
+
+      // FIXME: workaround for https://github.com/redhat-developer/app-services-ui-components/issues/590
+      // https://github.com/microsoft/playwright/issues/15734#issuecomment-1188245775
+      await new Promise((resolve) => setTimeout(resolve, 500));
```

Comment:
Could be `sleep` used here instead? https://github.com/redhat-developer/consoledot-e2e/blob/main/lib/common.ts#L2

Reply:
This is how sleep should be used in java script according stack overflow

Reply:
correct, good catch